app/src/main/java/com/connorlay/knitgrid/fragments/PatternDetailFragment.java

```diff
@@ -41,6 +46,8 @@
 
     private PatternPresenter mPatternPresenter;
 
+    private RowColumnListener mListener;
+
     public static PatternDetailFragment newInstance(PatternPresenter patternPresenter) {
```

Comment:
We could define some kind of interface that encapsulates cell selection behavior. That way we can unit test it and the code in general will be easier to read.